src/toolbars/CutCopyPasteToolBar.cpp

```diff
@@ -0,0 +1,174 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  CutCopyPasteToolBar.cpp
+
+  ksoze95
+
+  See CutCopyPasteToolBar.h for details
+
+*******************************************************************//*!
+
+\class CutCopyPasteToolBar
+\brief A ToolBar that has the cut, copy and paste buttons on it.
+
+  This class, which is a child of Toolbar, creates the
+  window containing interfaces to cut/copy/paste
+  functions that are otherwise only available through
+  menus. The window can be embedded within a normal project
+  window, or within a ToolBarFrame.
+
+  All of the controls in this window were custom-written for
+  Audacity - they are not native controls on any platform -
+  however, it is intended that the images could be easily
+  replaced to allow "skinning" or just customization to
+  match the look and feel of each platform.
+
+*//*******************************************************************/
+
+
+
+#include "CutCopyPasteToolBar.h"
+
+// For compilers that support precompilation, includes "wx/wx.h".
+#include <wx/wxprec.h>
+
+#include <wx/setup.h> // for wxUSE_* macros
+
+#ifndef WX_PRECOMP
+#include <wx/app.h>
+#include <wx/event.h>
+#include <wx/image.h>
+#include <wx/intl.h>
+#include <wx/sizer.h>
+#include <wx/tooltip.h>
+#endif
+
+#include "AllThemeResources.h"
+#include "ImageManipulation.h"
+#include "../Menus.h"
+#include "Prefs.h"
+#include "Project.h"
+#include "UndoManager.h"
+#include "../widgets/AButton.h"
+
+#include "../commands/CommandContext.h"
+#include "../commands/CommandManager.h"
+#include "../commands/CommandDispatch.h"
+
+enum {
+   TBCutID,
+   TBCopyID,
+   TBPasteID,
+   TBNumButtons
+};
+
+constexpr int first_TB_ID = 21300;
+
+static const ButtonList CutCopyPasteToolbarButtonList = {
+   { TBCutID,   wxT("Cut"),   XO("Cut")   },
+   { TBCopyID,  wxT("Copy"),  XO("Copy")  },
+   { TBPasteID, wxT("Paste"), XO("Paste") }
+};
+
+IMPLEMENT_CLASS(CutCopyPasteToolBar, ToolBar);
+
+////////////////////////////////////////////////////////////
+/// Methods for CutCopyPasteToolBar
+////////////////////////////////////////////////////////////
+
+BEGIN_EVENT_TABLE( CutCopyPasteToolBar, ToolBar )
+   EVT_COMMAND_RANGE(TBCutID+first_TB_ID,
+                      TBCutID+first_TB_ID + TBNumButtons - 1,
+                      wxEVT_COMMAND_BUTTON_CLICKED,
+                      CutCopyPasteToolBar::OnButton )
+END_EVENT_TABLE()
+
+//Standard constructor
+CutCopyPasteToolBar::CutCopyPasteToolBar( AudacityProject &project )
+: ToolBar(project, CutCopyPasteBarID, XO("Cut/Copy/Paste"), wxT("Cut/Copy/Paste"))
```

Comment:
For the fourth argument -- an internal identifier the user doesn't see, which may be used as a key in the configuration file -- do not use the '/' character.  I think it might get misinterpreted as multiple config file path components.  Just omit those characters.